src/types/ros3d.d.ts

```diff
@@ -153,5 +143,6 @@ declare module 'ros3d' {
     })
   }
 
-  type COLLADA_LOADER = any
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface, @typescript-eslint/class-name-casing
+  class COLLADA_LOADER_2 {}
```

Comment:
There's a warning saying that COLLADA_LOADER_2 is never used.

Reply:
fixed